community/cypher/acceptance-spec-suite/src/test/resources/cypher/features/BuiltInProcedureAcceptance.feature

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2002-2016 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# This file is part of Neo4j.
+#
+# Neo4j is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+Feature: BuiltInProcedureAcceptance
+
+  Background:
+    Given an empty graph
+
+  Scenario: Getting all labels
+    And having executed:
+    """
+    CREATE (:A)
+    CREATE (:B)
+    CREATE (:C)
+    """
+    When executing query:
+    """
+    CALL db.labels
+    """
+    Then the result should be, in order:
+      | label |
```

Comment:
Will the column have this name by default? Do procedures declare default output columns?